assets/src/application/ui/forms/modalForm/styles.scss

```diff
@@ -1,3 +1,15 @@
 .ee-modal-form {
 	max-width: 75rem !important;
+	.confirm-close {
+		display: inline-flex;
+		position: absolute;
+		top: 8px;
+		right: 12px;
+		min-width: auto;
+		width: 2rem;
+		height: 2rem;
+		svg {
+			margin: 0;
+		}
```

Comment:
not :100: with this, will improve in followup PR

Reply:
plz create an issue and document what needs to change so that it is not forgotten about

Reply:
I will create another related PR, left a comment to the original issue: https://github.com/eventespresso/event-espresso-core/issues/2613#issuecomment-614534126